var/spack/repos/builtin/packages/hypre-cmake/package.py

```diff
@@ -92,38 +94,36 @@ def setup_build_environment(self, env):
 
     @run_after("install")
     def cache_test_sources(self):
-        self.cache_extra_test_sources(self.extra_install_tests)
+        cache_extra_test_sources(self, self.extra_install_tests)
 
     @property
     def _cached_tests_work_dir(self):
         """The working directory for cached test sources."""
         return join_path(self.test_suite.current_test_cache_dir, self.extra_install_tests)
 
-    def test(self):
-        """Perform smoke test on installed HYPRE package."""
+    def test_hypre_cmake(self):
```

Comment:
The following is more descriptive (in case they want to add more tests in the future):

```suggestion
    def test_bigint(self):
```